### PR TITLE
WT-12050 Add long_running label to csuite tests exceeding 20 minutes

### PR DESCRIPTION
**Pull request by ajmorton**
Tag the longer running csuite tests with `long_running` so they'll be excluded from `csuite-tests-pull-request`. This keeps the runtime of `csuite-tests-pull-request` under 20 minutes for fast PR testing turnaround.

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -2232,7 +2229,6 @@ tasks:
       - func: "compile wiredtiger"
       - func: "make check directory"
         vars:
-          # Run the first test, and then every second test
           extra_args: -L long_running
```
Comment (ajmorton) [inline test/evergreen.yml:2232]: Question: Is there a reason we don't run these csuite tests in parallel? It'd bring runtime down from ~1 hour to ~10 minutes

Comment (etienneptl) [reply test/evergreen.yml:2232]: I don't think there is any reason not to. Could we give it a go in a patch build?

Comment (ajmorton) [reply test/evergreen.yml]: There's nothing in the original PR to suggest issues and my patch passes with a 40 minute time save. I've added the parallel flag in [7a6fbaa](https://github.com/wiredtiger/wiredtiger/pull/9938/commits/7a6fbaacc5519c86f2d5a3c79b7b30687a7308a6)

**Comment by ajmorton** [comment]
evergreen merge

**Pull request merged**